_includes/_howto/deployment_prerequisites.md

```diff
@@ -39,10 +38,12 @@ To create a service account for Forseti Inventory, Scanner, and Enforcer, follow
       $ export GOOGLE_APPLICATION_CREDENTIALS=SERVICE_ACCOUNT_KEY_PATH
       ``` 
           
-  where `SERVICE_ACCOUNT_KEY_PATH` is the path to the json service account key you just downloaded.
+  where `SERVICE_ACCOUNT_KEY_PATH` is the path to the json service account key
```

Comment:
We don't need to use, thus download, the json key of the forseti GCP service account anymore.  On GCP, this service account will just use the environment permission.